website/addons/dryad/static/dryadLogActionList.json

```diff
@@ -0,0 +1,11 @@
+{
+    "dryad_doi_set" : "${user} linked the Dryad package ${doi} to ${node}",
```

Comment:
The variables available to these templates are set in `website/static/js/logTextParser.js`.  `${user}` and `${node}` should already be defined, but you will probably need to create a parser for `${doi}`.  For an example, see the [`${repo}` variable](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/addons/github/static/githubLogActionList.json#L2) in the GitHub log actions.  The code to support that is [here](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/js/logTextParser.js#L487) and relies on variables exposed through the Node Logs API [here](https://github.com/CenterForOpenScience/osf.io/blob/develop-backup/api/logs/serializers.py#L56).